engine/src/main/java/org/terasology/engine/network/internal/NetworkSystemImpl.java

```diff
@@ -972,7 +973,8 @@ private <T> Map<Class<? extends T>, Integer> generateIds(ClassLibrary<T> classLi
             int fieldId = 0;
             for (FieldMetadata<?, ?> field : metadata.getFields()) {
                 if (fieldId >= 256) {
-                    logger.error("Class {} has too many fields (>255), serialization will be incomplete", metadata.getId());
+                    logger.atError().addArgument(() -> metadata.getId()).
+                            log("Class {} has too many fields (>255), serialization will be incomplete");
```

Comment:
simple getter method and only called in unexpected case once so impact neglectable
```suggestion
                    logger.error("Class {} has too many fields (>255), serialization will be incomplete", metadata.getId()); //NOPMD
```